packages/itwinui-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -462,9 +462,7 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
           },
           // ComboBox.MenuItem handles scrollIntoView, data-iui-index and iui-focused through context
           // but we still need to pass them here for backwards compatibility with MenuItem
-          className: cx(customItem.props.className, {
-            'iui-focused': focusedIndex === __originalIndex,
-          }),
+          'data-iui-focused': focusedIndex === __originalIndex,
```

Comment:
Is this data attribute necessary? We might be able to use `focused` prop instead.

Reply:
using prop

Reply:
Updated 'iui-focused' to 'focused' in that comment